exchanges/bithumb/bithumb_wrapper.go

```diff
@@ -438,22 +438,31 @@ func (b *Bithumb) SubmitOrder(s *order.Submit) (order.SubmitResponse, error) {
 
 // ModifyOrder will allow of changing orderbook placement and limit to
 // market conversion
-func (b *Bithumb) ModifyOrder(action *order.Modify) (string, error) {
+func (b *Bithumb) ModifyOrder(action *order.Modify) (order.Modify, error) {
 	if err := action.Validate(); err != nil {
-		return "", err
+		return order.Modify{}, err
 	}
 
-	order, err := b.ModifyTrade(action.ID,
+	o, err := b.ModifyTrade(action.ID,
```

Comment:
You'll need to do a check here for the `side` parameter. Since your RPCServer function only modifies price and amount, it means the side will be unset if performed via GRPC. You could otherwise populate some fields inside `(m *OrderManager) Modify` when you retrieve the existing order from the order manager to ensure fields such as this are still populated.